testutil/daemon/daemon.go

```diff
@@ -382,6 +389,9 @@ func (d *Daemon) StartWithBusybox(t testing.TB, arg ...string) {
 // Kill will send a SIGKILL to the daemon
 func (d *Daemon) Kill() error {
 	if d.cmd == nil || d.Wait == nil {
+		if err := mount.Unmount(d.Root); err != nil {
+			d.log.Logf("[%s] failed to unmount daemon root (%s): %v", d.id, d.Root, err)
```

Comment:
Dropped this commit, because this would also unmount the daemon root if live-restore is enabled, probably causing this failure:

```
 === FAIL: amd64.integration-cli TestDockerDaemonSuite/TestExecWithUserAfterLiveRestore (2.90s)
     --- FAIL: TestDockerDaemonSuite/TestExecWithUserAfterLiveRestore (2.90s)
         daemon.go:26: Creating a new daemon at: "/go/src/github.com/docker/docker/bundles/test-integration/TestDockerDaemonSuite/TestExecWithUserAfterLiveRestore"
         docker_cli_daemon_test.go:2723: assertion failed: expression is false: err == nil: Output: unable to find user test: no matching entries in passwd file
```